### PR TITLE
New article: Sign Up for Rackspace Services 

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes issue #2034

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-503518b219/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-760bc8c7a2/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-12b7c9245d/how-to/). :bow:

File: content/general/sign-up-for-racksapce-services.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+When you have decided that Rackspace Fanatical Support&reg; is for you, it's time to sign up for a Rackspace account. The process of signing up for an account is simple. However, it’s important to understand the verification process and account ownership if you’re new Rackspace customer.
+
+### Before you sign up
+
+Before you sign up for a Rackspace account, ask yourself the following questions:
+
+**Which cloud or Rackspace product are you using?**
+
+Rackspace supports a several different clouds including Amazon Web Services and Microsoft Azure, as well as Rackspace’s own suite of cloud products including Rackspace Private Cloud and Rackspace Public Cloud. The sign up process changes depending on what product you are using. Use the following list to find the right sign up process for your chosen Rackspace product or service.
```
Comment (ReneeRendon) [inline content/general/sign-up-for-racksapce-services.md]: Should we say, "...depending on which product or products you are using." ?

Comment (ReneeRendon) [inline content/general/sign-up-for-racksapce-services.md]: And, "Use the following list to find the right sign up process for your chosen Rackspace products and services."

File: content/general/sign-up-for-racksapce-services.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+-	Fanatical Support for AWS, Azure, and Rackspace Public Cloud
+-	Rackspace Email and applications
+-	Office 365
+
+**Who will be the primary user/account owner?**
+
+The information provided to Rackspace during the signup process is associated with the primary user of the account. The primary user is the owner of the account and is the account user that has the ability to add and remove additional users, set permissions and view or change billing information. In addition, the primary user is the main point of contact for the account. For example, any user can create a support ticket, but only the account owner will be notified when a support ticket is created, updated or marked as solved.
```
Comment (ReneeRendon) [inline content/general/sign-up-for-racksapce-services.md]: "For example, any user can create a support ticket, but only the account owner will be notified when a support ticket is created, updated or marked as solved."

Did this recently change with the new ticketing system?

File: content/general/sign-up-for-racksapce-services.md (hunk 1 above)
Comment (ReneeRendon) [inline content/general/sign-up-for-racksapce-services.md]: Remove "a".

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d0df7d8014/how-to/).

**Comment by DonNateR** [comment]
Waiting for SME review.

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits attached. Please let me know if you have any questions. Thanks!

[Sign up for Rackspace services_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1106682/Sign.up.for.Rackspace.services_editsKH.pdf)
[Sign up for Rackspace services_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1106683/Sign.up.for.Rackspace.services_editsKH.docx)

File: content/general/sign-up-for-racksapce-services.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+type: article
+created_date: '2017-06-15'
+created_by: Nate Archer
+last_modified_date: '2017-06-15'
+last_modified_by: Nate Archer
+product: undefined
+product_url: undefined
+---
+
+When you have decide that Rackspace Fanatical Support&reg;  is for you, it's time to sign up for a Rackspace account. Although the signup process is straightforward, it’s important to understand some details about the verification process and account ownership before you start.
```
Comment (stephamon) [inline content/general/sign-up-for-racksapce-services.md]: Looks like an extra space after the registered trademark

**Comment by stephamon** [comment]
👍

**Pull request merged**